detection.py

```diff
@@ -128,3 +130,41 @@ def detect_drift(
                     print(
                         f"Drift detected in feature {feature} before and after {time_cutoffs[ix]}"
                     )
+
+    def _remove_missing_values(
+        self,
+        baseline: Union[list, np.ndarray, pd.Series],
+        new: Union[list, np.ndarray, pd.Series],
+        base_cat_freq: OrderedDict,
+        new_cat_freq: OrderedDict
+    ) -> Tuple[OrderedDict, OrderedDict]:
+        """
+        This method gets the missing values frequencies from the original arrays and compares missing values frequencies
+        :param baseline: Baseline (old period / train set) vector
+        :param new: Target (future period / test set) vector
+        :param base_cat_freq: Baseline vector value counts with missing values
+        :param new_cat_freq: Target vector value counts with missing values
+        """
+        # Transform inputs into pandas series
+        baseline = pd.Series(baseline)
+        new = pd.Series(new)
+
+        # Calculate proportions of missing values
+        base_missing_prop = baseline.isna().mean()
+        new_missing_prop = new.isna().mean()
+
+        # Validate missing values proportions
+        if base_missing_prop != 0 and new_missing_prop != 0 and \
+                (new_missing_prop > base_missing_prop * (1 + self._significance) or
+                 base_missing_prop > new_missing_prop * (1 + self._significance)):
+            raise ValueError("Too many missing values")
```

Comment:
We don't want to raise errors here, we just want to alert the user about the drift